.github/workflows/CICD.yml

```diff
@@ -956,7 +956,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
You removed `windows-latest` on the wrong job ;-)

Reply:
oups :)